pkg/kubelet/kubelet.go

```diff
@@ -328,6 +328,43 @@ func getRuntimeAndImageServices(remoteRuntimeEndpoint string, remoteImageEndpoin
 	return rs, is, err
 }
 
+// queryVariablesFromCloud query nodeName, cloudIPs, cloudNames from cloud, these params are both in and out params
+func queryVariablesFromCloud(cloud cloudprovider.Interface, hostName string, nodeName *types.NodeName, cloudIPs *[]net.IP, cloudNames *[]string) (err error) {
```

Comment:
The in/out parameters aren't a common pattern used in Kubernetes. We should use multiple return parameters instead. 

`queryVariablesFromCloud` is a little vague for a function name.